.gitignore

```diff
@@ -11,52 +9,38 @@
 
 # vs code
 .vscode/
+*.code-workspace
+.unititest_domain_home/
 
 casual.env
-
-.unittest_domain_home/
-
-
 core
 
 
-#
 # Build directories
-#
 dependencies/
 obj/
 bin/
 
-#
 # Build files
-#
-*.mk
 *.pyc
-
 *.log
 /Debug/
 
-
-#
 # casual stuff
-#
 .casual/
 
-
 /Default/
 /Release/
 
-#
-# Tag files produced by Global
-#
+# Tag files produced by Global (what is this?)
 GPATH
```

Comment:
Googlade. Fick up:
GNU Global is a source code tagging system that works the same way across diverse environments, such as Emacs editor, Vi editor, Less viewer, Bash shell, ...

Detta stämmer också med min minnesbild. Tyckte mig känna igen GTAGS med flera i ett Emacs context.

Reply:
Thanks